prime-router/src/main/kotlin/fhirengine/engine/FHIRTranslator.kt

```diff
@@ -61,8 +61,8 @@ class FHIRTranslator(
                 // We only process receivers that are active and for this pipeline.
                 if (receiver != null && receiver.topic == Topic.FULL_ELR.json_val) {
                     val converter = FhirToHl7Converter(
-                        "ORU_R01-base",
-                        "metadata/hl7_mapping/ORU_R01"
+                        receiver.schemaName,
```

Comment:
You will need to add the new constructor I added in my play branch and the check as well.  Check the code I mentioned a while back